docker/worker/Dockerfile

```diff
@@ -1,21 +1,20 @@
+#!BuildTag: openqa_worker
 FROM opensuse/leap:15.2
 LABEL maintainer Jan Sedlak <jsedlak@redhat.com>, Josef Skladanka <jskladan@redhat.com>, wnereiz <wnereiz@eienteiland.org>, Sergio Lindo Mansilla <slindomansilla@suse.com>
 LABEL version="0.3"
 
-RUN zypper ar -f obs://devel:openQA/openSUSE_Leap_15.2 openQA && \
-    zypper ar -f obs://devel:openQA:Leap:15.2/openSUSE_Leap_15.2 openQA-perl-modules && \
+RUN zypper ar -p 95 -f http://download.opensuse.org/repositories/devel:openQA/openSUSE_Leap_15.2 devel_openQA && \
+    zypper ar -p 90 -f http://download.opensuse.org/repositories/devel:openQA:Leap:15.2/openSUSE_Leap_15.2 devel_openQA_Leap && \
     zypper --gpg-auto-import-keys ref && \
     zypper --non-interactive in ca-certificates-mozilla curl gzip && \
     zypper --non-interactive in openQA-worker qemu-arm qemu-ppc qemu-x86 && \
     zypper --non-interactive in kmod && \
-    (zypper --non-interactive in qemu-ovmf-x86_64 || true) && \
-    (zypper --non-interactive in qemu-uefi-aarch64 || true)
+    zypper --non-interactive in qemu-ovmf-x86_64
 
-# set-up qemu
 RUN mkdir -p /root/qemu
 ADD kvm-mknod.sh /root/qemu/kvm-mknod.sh
 ADD run_openqa_worker.sh /run_openqa_worker.sh
-RUN chmod +x /root/qemu/kvm-mknod.sh && /root/qemu/kvm-mknod.sh && \
+RUN chmod +x /root/qemu/kvm-mknod.sh && \
```

Comment:
> OK. Can you please add this as a comment?

You didn't add a comment? 🤔 But I see this in the commit message:

> kvm operations cannot be instanced during the building
because the workers may have kvm but the build machine don't
have to

Reply:
Ok. done